internal/kitchensinktest/options.go

```diff
@@ -56,6 +62,9 @@ type Options struct {
 	// used over the deprecated EndpointResolver.
 	EndpointResolverV2 EndpointResolverV2
 
+	// Signature Version 4 (SigV4) Signer
+	HTTPSignerV4 HTTPSignerV4
```

Comment:
Do we actually need to have a full auth scheme and v4 signer to test credentials refresh? Would this still work if we had static credentials?

Reply:
There's no credentials provider in options without sigv4 or sigv4a (note how the diff shows it was added here)